modules/react/popup/spec/Popper.spec.tsx

```diff
@@ -84,15 +84,15 @@ describe('Popper', () => {
 
     // force PopperJS to run
     // eslint-disable-next-line compat/compat
-    await act(() => new Promise(requestAnimationFrame));
+    await act(() => new Promise<any>(requestAnimationFrame));
```

Comment:
Why is this `any` needed? It should extract the type from `requestAnimationFrame`, but we don't actually care about the return type.

Reply:
ReactDOM test utils changed the type of the promise that could be returned to `act` and forced it to be `void` even though the promise return type is ignored. `requestAnimationFrame` takes a callback that expects a `number` and they 2 come into conflict.

The alternative is:
```ts
await act(() => new Promise(cb => requestAnimationFrame(() => cb())));
```

But that seems messier